graphql/resolvers/tasks.js

```diff
@@ -0,0 +1,37 @@
+const Task = require("../../models/task");
+const moment = require('moment');
+
+module.exports = {
+  tasks: async (args) => {
+    try {
+      let tasks = await Task.find({ userId: args.userId }).populate(
+        "assignedBy"
+      );
+      return tasks.map((result) => {
```

Comment:
instead of mapping of result you can use .select() in express

Reply:
Resolved